shared/tracker/render.js.flow

```diff
@@ -7,7 +7,7 @@ import type {ActionProps} from './action.render'
 import type {HeaderProps} from './header.render'
 import type {ProofsProps} from './proofs.render'
 
-export type RenderProps = $Shape<{
+export type RenderPropsUnshaped = {
```

Comment:
Without this we get an error when we include parentProps in the `trackerPropsToRenderProps` fn